From 639310e8345c617ad4c5b7f93131aced5c4036e7 Mon Sep 17 00:00:00 2001
From: Lim Siew Hoon <siew.hoon.lim@intel.com>
Date: Tue, 8 Aug 2023 14:42:11 +0800
Subject: [PATCH 1/3] Change version to 1.22.4

No different between code based 1.22.3 vs 1.22.3.
But we required it inside 1.22.4 version, since all
others gstreamer components move to 1.22.4 version.

Signed-off-by: Lim Siew Hoon <siew.hoon.lim@intel.com>
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index b9d7d0d..5c2c354 100644
--- a/meson.build
+++ b/meson.build
@@ -1,5 +1,5 @@
 project('gst-rtsp-server', 'c',
-  version : '1.22.3',
+  version : '1.22.4',
   meson_version : '>= 0.62',
   default_options : ['warning_level=1', 'buildtype=debugoptimized'])
 
-- 
2.40.1

